packages/react-data-grid/src/common/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface Column {
```

Comment:
https://github.com/adazzle/react-data-grid/pull/1561/files#diff-17633e7ac6651e7fd23ad88c079c43efR3

It's duplicated here.

Reply:
Resolved.